subt/docker/robotika/run_solution.bash

```diff
@@ -26,7 +26,7 @@ echo "Robot name is '$ROBOT_NAME'"
 
 # Run your solution and wait for ROS master
 # http://wiki.ros.org/roslaunch/Commandline%20Tools#line-45
-roslaunch subt_seed x1.launch --wait robot_name:=$ROBOT_NAME &
+roslaunch robotika robot.launch --wait robot_name:=$ROBOT_NAME &
```

Comment:
I think we somehow have to find a way how to say if we are launching x2 or x4. I think we can get this information using `rosparam` before we actually launch anything and decide based on that.

Reply:
Ok, when we have robot name, we can also do `rosparam get /<robot_name>/robot_description` which contains xml from robot directory - for our X2 it says it contains file `/home/developer/subt_ws/install/share/robotika_x2_sensor_config_1/urdf/robot_from_sdf.xacro`. I think just grepping that mess for `ROBOTIKA_X2` should be enough (it has some very very weird escaping which so far eludes me).

Reply:
I'm not sure with this solution. It will help short term, but the primary problem is not that we want to distinguish configs for different robots (e.g. X2 or X4), but we want to distinguish configurations which how we run a specific robot. E.g. for the Urban, I was running X2 with additonal ros modules (it means package "robotika", and robot.launch).

Reply:
I will find out how to distinguish urban from cave as well.

Reply:
<pre>developer@larsa:~/subt_ws$ rosparam get world_name
urban_circuit_practice_01
</pre>

So checking for `tunnel`, `urban` and `cave` might work. For the finals I am afraid we will have to do without this information anyway.